src/useValue.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import AnimatedValue from './core/AnimatedValue';
+
+export default function useValue(initialValue) {
+  return React.useRef(new AnimatedValue(initialValue)).current;
```

Comment:
When rerendering functional component `useValue` is called and we create `new AnimatedValue(initialValue)`, don't we?

WDYT about this impl?
```js
export default function useValue(initialValue) {
  const ref = React.useRef(null);
  if(ref === null){
    ref.current = new AnimatedValue(initialValue);
  }
  return ref.current
}
```

Reply:
I'm totally fine with this 👍If creating new instance could lead to perf issues let's use this one

Reply:
just change `if(ref === null){` to `if(ref.current === null){` 🙂

Reply:
You're right, should be `ref.current`

Reanimated have some issues with initialization of nodes, so we aim to minimize nodes created whenever possible 🙂

Reply:
done ✔️